nixos/modules/services/networking/networkmanager.nix

```diff
@@ -227,13 +215,30 @@ in {
         '';
       };
 
-      packages = mkOption {
+      basePackages = mkOption {
+        type = types.listOf types.package;
+        default = with pkgs; [
+          modemmanager
+          networkmanager
```

Comment:
I think `networkmanager` should just always be included? Does it make sense to enable the module without including the base package?

Reply:
maybe someone wants to override the package? but yeah, maybe we should do it some other way, but this was the least complex way I could think about